scripts/create_raid0.sh

```diff
@@ -12,6 +18,57 @@ while (( "$#" )); do
     shift
 done
 
+# Check to see which OS this is running on. 
+os_release=$(cat /etc/os-release | grep "^ID\=" | cut -d'=' -f 2 | sed -e 's/^"//' -e 's/"$//')
```

Comment:
It would be better to include a library script with a function to return the OS versions or with helpers like is_ubuntu or is_centos instead of duplicating all these lines in plenty of scripts. This will make it easier for maintenance. call the include script at the beginning and add it as a dependency in the config files where the script is used.